product_offer/views/product_template_views.xml

```diff
@@ -58,9 +58,22 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='product_variant_ids']" position="before">
                 <separator/>
-                <filter name="stock_available_oaw" string="Available (Local/Overseas)" domain="['|',('local_stock','=','Yes'),('overseas_stock','=','Yes')]"/>
-                <filter name="received_two_weeks" string="Received in the Last Two Weeks" domain="[('last_in_date','&gt;=',(context_today()+datetime.timedelta(weeks=-2)).strftime('%Y-%m-%d'))]"/>
+
+                <filter name="received_two_weeks" string="Received in the Last Two Weeks" domain="[('last_in_date','&gt;=',(context_today()+datetime.timedelta(weeks=-2)).strftime('%Y-%m-%d'))]"/><br/>
                 <field name="supplier_id"/>
+                <group expand="0" string="Available">
+
+                    <filter name="local_w_res" string="Local w R" domain="[('qty_local_stock','&gt;',0)]"/>
+
+                    <filter name="local_wo_res" string="Local w/o R" domain="[('local_stock','=','Yes')]"/>
+
+                     <filter name="overseas" string="Overseas" domain="[('overseas_stock','=','Yes')]"/>
```

Comment:
I'd make the domain domain="[('qty_overseas','&gt;',0')]" to slightly lower the dependency on the customized logic. But the way you do is also OK.